zipline/finance/risk/report.py

```diff
@@ -0,0 +1,153 @@
+    | key             | value                                              |
+    +=================+====================================================+
+    | trading_days    | The number of trading days between self.start_date |
+    |                 | and self.end_date                                  |
+    +-----------------+----------------------------------------------------+
+    | benchmark_volat\| The volatility of the benchmark between            |
+    | ility           | self.start_date and self.end_date.                 |
+    +-----------------+----------------------------------------------------+
+    | algo_volatility | The volatility of the algo between self.start_date |
+    |                 | and self.end_date.                                 |
+    +-----------------+----------------------------------------------------+
+    | treasury_period\| The return of treasuries over the period. Treasury |
+    | _return         | maturity is chosen to match the duration of the    |
+    |                 | test period.                                       |
+    +-----------------+----------------------------------------------------+
+    | sharpe          | The sharpe ratio based on the _algorithm_ (rather  |
+    |                 | than the static portfolio) returns.                |
+    +-----------------+----------------------------------------------------+
+    | information     | The information ratio based on the _algorithm_     |
+    |                 | (rather than the static portfolio) returns.        |
+    +-----------------+----------------------------------------------------+
+    | beta            | The _algorithm_ beta to the benchmark.             |
+    +-----------------+----------------------------------------------------+
+    | alpha           | The _algorithm_ alpha to the benchmark.            |
+    +-----------------+----------------------------------------------------+
+    | excess_return   | The excess return of the algorithm over the        |
+    |                 | treasuries.                                        |
+    +-----------------+----------------------------------------------------+
+    | max_drawdown    | The largest relative peak to relative trough move  |
+    |                 | for the portfolio returns between self.start_date  |
+    |                 | and self.end_date.                                 |
+    +-----------------+----------------------------------------------------+
+    | max_leverage    | The largest gross leverage between self.start_date |
+    |                 | and self.end_date                                  |
+    +-----------------+----------------------------------------------------+
+
+"""
+
+import logbook
+import datetime
+from dateutil.relativedelta import relativedelta
+
+from . period import RiskMetricsPeriod
+
+log = logbook.Logger('Risk Report')
+
+
+class RiskReport(object):
```

Comment:
This class is also just a function, it only exists to forward args to `to_dict`.